ui/store/actions.ts

```diff
@@ -5429,18 +5429,17 @@ export function updateOnChainTriggersByAccount(
  * Upon success, it dispatches an action with type `FETCH_AND_UPDATE_METAMASK_NOTIFICATIONS` to update the Redux state.
```

Comment:
Nit (future) - can we create a ticket to correct these JSDOC comments?

We don't dispatch any type here. We did in the original implementation ages ago, but not anymore.

Reply:
@Prithpal-Sooriya I’ve at least updated the action I modified